build.gradle

```diff
@@ -109,7 +109,7 @@ subprojects {
 
 ext {
     // NB: due to version.json generation by parsing this file, the next line must not have any if/then/else logic
-    neo4jVersion = "4.4.1"
+    neo4jVersion = "4.4.4-SNAPSHOT"
```

Comment:
A new 4.4 version of the database needs to be released and substituted here.

Reply:
@ncordon If we don't use a non snapshot version we aren't able to test the PR in GH can you please change it to 4.4.3?

Reply:
The code we need is in 4.4.5, and to release 4.4.5 we are going to need this to be merged on red. The CI of the database will compile the APOC code overriding the version.

So even bumping this, don't expect the CI to pass.

After the database has been released, we can bump here to 4.4.5 and release a new version of APOC.

Reply:
This was done without the need to depend on 4.4.4 in the end